packages/shared/user-data/GraphQL/config.ts

```diff
@@ -43,6 +43,12 @@ export const config = {
     label: "Disable using incremental snapshots",
     legacyKey: "devtools.features.disableIncrementalSnapshots",
   },
+  backend_enableHasAnnotationKindQueryStorage: {
+    defaultValue: Boolean(false),
+    description: "Enable storage of previously generated response to Session.hasAnnotationKind",
+    label: "Enable query-level storage for Session.hasAnnotationKind",
```

Comment:
this is a bit specific for a user-facing experimental setting right?

Reply:
The label and description are what get displayed in the preferences dialog, yes. Whether or not they show up in "experimental" just depends on where you reference them. (This PR does put them in experimental though, assuming that's what you want.)

It's not clear to me if this should be an internal only setting.

Reply:
Hmm I think it might need to be an internal only setting for now. Did something change recently about how to set those settings from the console? I was having a hard time figuring that out yesterday.

Reply:
The config objects support an optional `internalOnly` which are automatically hidden from non-Replay users in the preferences panel.

Reply:
Thanks! Fixed